.storybook/YourTheme.ts

```diff
@@ -0,0 +1,41 @@
+// .storybook/YourTheme.ts
+import { create } from '@storybook/theming/create';
+import logo from '../public/assets/freenow-logo.svg';
+
+export default create({
+    base: 'light',
+
+    // Typography
+    fontBase: '"YOURNOWSans-Medium", "Open Sans", sans-serif',
```

Comment:
we need to serve the custom font, but I don't see it imported in the styled. I think we should bring the font file here and write `font-family` declaration

Reply:
Note: Adding `YOURNOWSans-Medium` as a font-family for theme was a mistake (due to me using it in Adobe AI, when generating the SVG for ''by Freenow" part in logo), it should be and will remain the default font of Wave DS which is `Open Sans` .